tests/snmpsim/netscaler-sdx.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.5951.6
```

Comment:
Ok. Done. The real hostname didn't even have a domain in the snmpwalk. I've just replaced it with "examplehost".